Makefile

```diff
@@ -1,4 +1,5 @@
 export DJANGO_SETTINGS_MODULE=eventtracking.django.tests.settings
+export PYTHONPATH=$(PWD)
```

Comment:
This strikes me as odd. I don't think you can assume that everyone's python path will be at the root of this directory. Also isn't . usually on the python path? I don't think this should be here.

Reply:
Removed this by changing `pytest` command to `python -m pytest`